### PR TITLE
Add Pypi feed testing

### PR DESCRIPTION
**Pull request by Qinusty**
Adds some tests to the pypi feed using httptest as a mock endpoint.

### CODE REVIEW

File: feeds/pypi/pypi.go [hunk 1]
```diff
@@ -69,11 +68,19 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
 }
 
-type Feed struct{}
+type Feed struct {
```
Comment (tom--pollard) [inline feeds/pypi/pypi.go:71]: I think the addition of the `Feed` struct to abstract the url is worthy of being a separate commit, but that's only a preference

**Comment by tom--pollard** [comment]
+1 for more testing!

File: feeds/pypi/pypi.go (hunk 1 above)
Comment (Qinusty) [reply feeds/pypi/pypi.go:71]: Agreed :+1:

**Comment by Qinusty** [comment]
Closing as this will be added to https://github.com/ossf/package-feeds/pull/76

**Pull request closed without merging**